mltsp/TCP/tests/test_feature_generation.py

```diff
@@ -10,58 +11,743 @@
 import glob
 
 
-def setup():
-    print("Copying data files")
-    # copy data files to proper directory:
-    shutil.copy(os.path.join(os.path.dirname(__file__),
-                             "data/asas_training_subset_classes.dat"),
-                os.path.join(cfg.UPLOAD_FOLDER))
-
-    shutil.copy(os.path.join(os.path.dirname(__file__),
-                             "data/asas_training_subset.tar.gz"),
-                os.path.join(cfg.UPLOAD_FOLDER))
-
-
-def test_feature_generation():
-    for f in glob.glob(os.path.join(cfg.FEATURES_FOLDER, '/*.csv')):
-        os.remove(f)
-
-    tic = time.time()
-    this_dir = os.path.join(os.path.dirname(__file__))
-
-    featurize.featurize(
-        os.path.join(cfg.UPLOAD_FOLDER, "asas_training_subset_classes.dat"),
-        os.path.join(cfg.UPLOAD_FOLDER, "asas_training_subset.tar.gz"),
-        featureset_id="testfeatset", is_test=True,
-        features_to_use=cfg.features_list_science
-    )
-
-    delta = time.time() - tic
-
-    def features_from_csv(filename):
-        with open(filename) as f:
-            feature_names = f.readline().strip().split(",")
-            feature_values = np.loadtxt(f, delimiter=',')
-
-        return feature_names, feature_values
-
-    features_extracted, values_computed = features_from_csv(
-        os.path.join(cfg.FEATURES_FOLDER, "testfeatset_features.csv"))
-
-    features_expected, values_expected = features_from_csv(
-        os.path.join(this_dir, "data/expected_features.csv"))
-
-    os.remove(os.path.join(cfg.FEATURES_FOLDER, "testfeatset_features.csv"))
-    os.remove(os.path.join(cfg.FEATURES_FOLDER,
-                           "testfeatset_classes.npy"))
-    npt.assert_equal(len(features_extracted), 81)
-    npt.assert_equal(features_extracted, features_expected)
-    npt.assert_array_almost_equal(values_computed, values_expected)
-
-    # Ensure this test takes less than a minute to run
-    assert delta < 60
-
-
-if __name__ == "__main__":
-    setup()
-    test_feature_generation()
+## TODO just a list for reference, will eventually remove
+#untested_extractors = [
+#                       'freq_model_phi1_phi2_extractor', # TODO wrong?
+#                       'freq_n_alias_extractor', # TODO what is this?
```

Comment:
These are the TCP tests that were deleted and then un-deleted; they had just been commented out because I changed some feature names and didn't want to fix them in the new and old places. This is basically a duplicate of `mltsp.science_features.tests.test_feature_generation`, just in the old `extractor.extract()` syntax.